drivers/entropy/entropy_npcx_drbg.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_npcx_drbg
+
+#include <errno.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/entropy.h>
+#include <zephyr/pm/device.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(entropy_npcx_drbg, CONFIG_ENTROPY_LOG_LEVEL);
+
+#include "soc_ncl.h"
+
+/* Reseed after 100 number generations */
+#define NPCX_DRBG_SECURITY_STRENGTH                                                                \
+	(enum ncl_drbg_security_strength) CONFIG_ENTROPY_NPCX_DRBG_SECURITY_STRENGTH
```

Comment:
should this have an extra parenthesis?

`((enum ncl_drbg_security_strength)CONFIG_ENTROPY_NPCX_DRBG_SECURITY_STRENGTH)`

Reply:
Done.